libstuff/SDeburr.cpp

```diff
@@ -0,0 +1,137 @@
+#include "SDeburr.h"
+
+#include <bit>
+#include <cstring>
+#include <string>
+
+#include <libstuff/libstuff.h>
+#include <libstuff/sqlite3.h>
+
+using namespace std;
+
+const char* SDeburr::unicodeToAscii(uint32_t codepoint) {
+    if (codepoint >= 0x00C0 && codepoint <= 0x017F) {
```

Comment:
Can you comment what these indicate? Possibly with a link to the unicode spec or something.